bazel_tools/BUILD.bazel

```diff
@@ -16,3 +19,65 @@ exports_files(
     ],
     visibility = ["//:__subpackages__"],
 )
+
+sh_inline_test(
```

Comment:
@cocreature, @aherrmann-da, I added a test case to check that the versions will all line up. Could you take a look please?